server_apps/jenkins/jobs/Load-Complete-Author-Names_d/config.xml

```diff
@@ -17,6 +17,12 @@
     <concurrentBuild>false</concurrentBuild>
     <customWorkspace>$TARGETROOT</customWorkspace>
     <builders>
+        <hudson.tasks.Shell>
+            <command><![CDATA[
```

Comment:
My preference would be to create that 'target'-type of directory from Java so that the execution is less dependent on the environment in which it is executed. Applies to the following similar changes as well.

Reply:
@cmpich , Good point. I removed this type of logic from the xml and added it to java or shell scripts where necessary.